internal/prefixcollector/collector.go

```diff
@@ -116,8 +119,23 @@ func (epc *ExcludedPrefixCollector) Serve(ctx context.Context) {
 	}
 }
 
+// splitPrefix splits single prefix string into list of prefixes treating the input as comma separated.
+// When cluster supports both IPv4 and IPv6 we can receive combined addresses e.g. "10.244.0.0/16,fd00:10:244::/56"
```

Comment:
Can we fix the problem for kubeadm source? Have a look at https://github.com/networkservicemesh/cmd-exclude-prefixes-k8s/blob/main/internal/prefixcollector/prefixsource/kubeadm_prefix_source.go#L146